src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.X509.cs

```diff
@@ -22,20 +22,25 @@ internal static byte[] X509Encode(SafeX509Handle x)
         }
 
         [DllImport(Libraries.CryptoNative, EntryPoint = "AndroidCryptoNative_X509DecodeCollection")]
-        private static extern int X509DecodeCollection(ref byte buf, int bufLen, IntPtr[]? ptrs, int handlesLen);
+        private static extern int X509DecodeCollection(ref byte buf, int bufLen, IntPtr[]? ptrs, out int handlesLen);
```

Comment:
```suggestion
        private static extern int X509DecodeCollection(ref byte buf, int bufLen, IntPtr[]? ptrs, ref int handlesLen);
```

The native code is reading this value, so it should be `ref` instead of `out`.